pegasus/sites.v3/hourofcode.com/public/css/front-page.scss

```diff
@@ -99,7 +99,6 @@ section.join-us {
   .row {
 
     & > div:first-child {
-      float: none;
```

Comment:
I removed this line to float the signup form to the left.

Reply:
This seems to not affect other parts of the page, but it seems risky –– I can see the `.row` class from `home_signup.haml`, but it's such a not-specific name that I worry other elements on the page would be affected. It doesn't look like that's the case, but I'd be in favor of adding a more specific classname for the future (in case there become more .row classes on this page)

Reply:
It's nested in the `section.join-us` section so it shouldn't affect anything else on the page, but I agree it could be part of a future refactor. https://github.com/code-dot-org/code-dot-org/blob/19bfffa35c9ce77653ed33f1005f136029766d0d/pegasus/sites.v3/hourofcode.com/public/css/front-page.scss#L91-L102

Reply:
Ah good call –– I missed that!